test/xpu_api/random/interface_tests/engines_methods.pass.cpp

```diff
@@ -1,461 +1,461 @@
+            Engine engine;
+            sum += check_params(engine);
+        }
+
+        return sum;
+    }
+};
+
+template <>
+class
+test<oneapi::dpl::ranlux24>
+{
+public:
+    bool run(sycl::queue& queue)
+    {
+        using result_type = typename oneapi::dpl::ranlux24::scalar_type;
+
+        int sum = 0;
+
+        // Memory allocation
+        std::vector<std::int32_t> dpstd_res(N_GEN);
+
+        // Random number generation
+        {
+            sycl::buffer<std::int32_t> dpstd_buffer(dpstd_res.data(), dpstd_res.size());
+
+            try
+            {
+                queue.submit([&](sycl::handler& cgh) {
+                    auto dpstd_acc = dpstd_buffer.template get_access<sycl::access::mode::write>(cgh);
+
+                    cgh.parallel_for<>(sycl::range<1>(N_GEN), [=](sycl::item<1> idx) {
+                        unsigned long long offset = idx.get_linear_id();
+                        oneapi::dpl::ranlux24 engine0(SEED);
+                        oneapi::dpl::ranlux24 engine1;
+                        engine1.seed(SEED);
+                        engine0.discard(offset);
+                        engine1.discard(offset);
+                        typename oneapi::dpl::ranlux24::result_type res0;
+                        oneapi::dpl::ranlux24 engine(engine1);
+                        auto eng = engine.base();
+                        res0 = engine();
+                        typename oneapi::dpl::ranlux24::result_type res1 = engine1();
+                        if (res0 != res1)
+                        {
+                            dpstd_acc[offset] = 1;
+                        }
+                        else
+                        {
+                            dpstd_acc[offset] = 0;
+                        }
+                    });
+                });
+                auto dpstd_acc = dpstd_buffer.template get_access<sycl::access::mode::write>();
+                for (int i = 0; i < N_GEN; ++i)
+                {
+                    sum += dpstd_acc[i];
+                }
+                if (sum)
+                {
+                    std::cout << "Error occurred in " << sum << " elements" << std::endl;
+                }
+            }
+            catch (sycl::exception const& e)
+            {
+                std::cout << "\t\tSYCL exception during generation\n"
+                          << e.what() << std::endl
+                          << "OpenCL status: " << e.get_cl_code() << std::endl;
+                return 1;
+            }
+
+            queue.wait_and_throw();
+            oneapi::dpl::ranlux24 engine;
+            sum += check_params(engine);
+        }
+
+        return sum;
+    }
+};
+
+#endif // TEST_DPCPP_BACKEND_PRESENT && TEST_UNNAMED_LAMBDAS
+
+int
+main()
+{
+
+#if TEST_DPCPP_BACKEND_PRESENT && TEST_UNNAMED_LAMBDAS
+    // Catch asynchronous exceptions
+    auto exception_handler = [](sycl::exception_list exceptions) {
+        for (std::exception_ptr const& e : exceptions)
```

Comment:
Do we expect such formatting here?

Reply:
Yes, we don't need to apply clang-format to tests and I didn't change formatting (only line endings and file permissions)